### ISSUE TITLE
About Tanks ang Temples

### ISSUE THREAD

**Issue by 123xju** [TQTQliu/ET-MVSNet#22]
Hello, thank you very much for your excellent work, I recently reproduced your work following the tutorial you posted, but there is a little problem, I need your help.
1. The quantitative results in the DTU dataset are basically the same as the results presented in your paper, but the evaluation effect in the TNT dataset is very poor, not even half of the data you present, I would like to ask if there is any good solution. I would appreciate it if you could tell me how
2. Is it possible that there is a problem with my steps, I will now roughly describe it, you see if it is appropriate. Firstly, the DTU dataset is used for training, the trained model is put into the blended dataset for fine-tuning to obtain the blended training model, and finally the blended training model is used as a checkpoint to test the TNT dataset.
![ff88a47734905d0567863471ffb029d](https://github.com/TQTQliu/ET-MVSNet/assets/174929639/5b30ec8f-3f65-4026-ae7e-88430f60b099)

**Comment by TQTQliu** [comment]
Hello, your training or fine-tuning steps are correct.
Due to the limited information provided so far, I am not sure what the specific problem is. 
However, the result of this anomaly is likely to be a problem with the data set, you can check it and download the data set we provided [here](https://github.com/TQTQliu/ET-MVSNet/blob/e29d0b6c1d7772a86b730f4b8b106df3846a7fb2/README.md?plain=1#L75).
And you can first use the [checkpoint](https://drive.google.com/drive/folders/1sXAR8Y91oahH0n3dgpJDT51i8mGr6rtZ?usp=drive_link) we provided to test whether you could get reasonable TNT quantitative results.

**Comment by 123xju** [comment]
I just need to change the TNT test set, or all the data sets, my DTU evaluation is similar to yours



---Original---
From: "Tianqi ***@***.***&gt;
Date: Sun, Jul 7, 2024 16:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About Tanks ang Temples (Issue #22)




 
Hello, your training or fine-tuning steps are correct.
 Due to the limited information provided so far, I am not sure what the specific problem is.
 However, the result of this anomaly is likely to be a problem with the data set, you can check it and download the data set we provided here.
 And you can first use the checkpoint we provided to test whether you could get reasonable TNT quantitative results.

**Comment by 05063112lcs** [comment]
Hello author, I'm sorry to bother you again, I followed the method you provided, used the dataset you provided, and even used the pre-trained model you provided, and still did not get effective visualization results, I have not yet evaluated. Is there any other solution, could it be a problem with the environment, the python version I use is 3.8.18

**Comment by 123xju** [comment]
Hello author, I'm sorry to bother you again, I followed the method you provided, used the dataset you provided, and even used the pre-trained model you provided, and still did not get effective visualization results, I have not yet evaluated. Is there any other solution, could it be a problem with the environment, the python version I use is 3.8.18

**Comment by TQTQliu** [comment]
Hello, can you show the visualization results?

**Comment by 05063112lcs** [comment]
Hello, thank you very much for replying to me again! According to your advice, I downloaded the TNT data set you recommended, and then tested it with the pre-training model you gave, but the effect was still poor, there was noise around the point cloud. Today, I saw that you mentioned the environment problem in your reply to others, I tried to reinstall the environment, and found that some packages you provided did not support the python version 3.10.8. Below is my point cloud visualization, tested with the training model you gave.



---Original---
From: "Tianqi ***@***.***&gt;
Date: Tue, Jul 9, 2024 13:36 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About Tanks ang Temples (Issue #22)




 
Hello, can you show the visualization results?

**Comment by 123xju** [comment]
Hello, thank you very much for replying to me again! According to your advice, I downloaded the TNT data set you recommended, and then tested it with the pre-training model you gave, but the effect was still poor, there was noise around the point cloud. Today, I saw that you mentioned the environment problem in your reply to others, I tried to reinstall the environment, and found that some packages you provided did not support the python version&nbsp;3.10.8. Below is my point cloud visualization, tested with the training model you gave.




---Original---
From: "Tianqi ***@***.***&gt;
Date: Tue, Jul 9, 2024 13:36 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About Tanks ang Temples (Issue #22)




 
Hello, can you show the visualization results?

**Comment by 123xju** [comment]
Hello, thank you very much for replying to me again! According to your advice, I downloaded the TNT data set you recommended, and then tested it with the pre-training model you gave, but the effect was still poor, there was noise around the point cloud. Today, I saw that you mentioned the environment problem in your reply to others, I tried to reinstall the environment, and found that some packages you provided did not support the python version&nbsp;3.10.8. Below is my point cloud visualization, tested with the training model you gave.
![image](https://github.com/TQTQliu/ET-MVSNet/assets/174929639/7b7fe014-e553-4afd-b5fd-0eced80c3c2c)

**Comment by TQTQliu** [comment]
Based on the visualizations presented, I think the results are reasonable and you can submit to get quantitative results. Looking forward to your feedback.

**Comment by 123xju** [comment]
Hello author I read your personal profile, very impressed, I am also a graduate, convenient to leave personal contact information, I may have a lot of questions to consult



---Original---
From: "Tianqi ***@***.***&gt;
Date: Tue, Jul 9, 2024 14:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About Tanks ang Temples (Issue #22)




 
Based on the visualizations presented, I think the results are reasonable and you can submit to get quantitative results. Looking forward to your feedback.

**Comment by 123xju** [comment]
Hi, this is my reproduction, using my own training model, slightly worse than the data in your paper. How can this final figure be the same or very close to yours



---Original---
From: "Tianqi ***@***.***&gt;
Date: Tue, Jul 9, 2024 14:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About Tanks ang Temples (Issue #22)




 
Based on the visualizations presented, I think the results are reasonable and you can submit to get quantitative results. Looking forward to your feedback.
